### PR TITLE
hiseq2500 fluidics update

### PR DESCRIPTION
**Pull request by ghost**
updates to config file and fluidics for HiSeq2500 support

### CODE REVIEW

**Comment by chaichontat** [comment]
Could you run `pre-commit run --all-files --show-diff-on-failure` on your PR? Need to add this to contribution.md

**Comment by krpandit** [comment]
Done.

File: pyseq2/fluidics/valve.py [hunk 1]
```diff
@@ -28,22 +28,27 @@ class ValveCmd:
 class _Valve(Movable, UsesSerial):
     @classmethod
     async def ainit(cls, name: ValveName, port_tx: str) -> _Valve:
-        n_ports = 24 if CONFIG.machine == "HiSeq2500" and name.startswith("valve_b") else 10
-        self = cls(name, n_ports)
```
Comment (chaichontat) [inline pyseq2/fluidics/valve.py:32]: This is a safety check in case the user puts in the wrong machine config

File: pyseq2/fluidics/valve.py [hunk 2]
```diff
@@ -107,8 +112,8 @@ async def pos(self) -> int:
             return p1
 
         elif CONFIG.machine == "HiSeq2500":
-            if p1 == 6:
-                return 0
```
Comment (chaichontat) [inline pyseq2/fluidics/valve.py:111]: Do you want to get rid of the safety ports or are there better ports that are unused.

File: pyseq2/fluidics/valve.py (hunk 1 above)
Comment (krpandit) [reply pyseq2/fluidics/valve.py:32]: its the 2nd valve that has 24 ports, so it should ```name[-1]  == '2'```

**Pull request merged**